index.js

```diff
@@ -20,6 +20,15 @@ function isObjectWithProperties(node) {
   return true;
 }
 
+// Default interface props don't need labels and descriptions
+function isDefaultInterfaceProperty(propertyName, properties) {
+  if (propertyName === "label" || propertyName === "description") {
+    const interfacePropValue = findPropertyWithName("type", properties).value;
+    return (interfacePropValue.value === "$.interface.timer" || interfacePropValue.value === "$.interface.http");
```

Comment:
I get the error `Cannot read property 'value' of undefined` when a prop is missing a `type` property. `findPropertyWithName("type", properties)` is `undefined`.

You could use the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator (`?.`) to return `undefined` instead of throwing an error when trying to access a property of `null` or `undefined`.

```js
    const interfacePropValue = findPropertyWithName("type", properties)?.value;
    return (interfacePropValue?.value === "$.interface.timer" || interfacePropValue?.value === "$.interface.http");
```
or 
```js
    const interfaceProp = findPropertyWithName("type", properties);
    const interfacePropValue = interfaceProp?.value.value;
    return (interfacePropValue === "$.interface.timer" || interfacePropValue === "$.interface.http");
```

**To reproduce error**
- In `tests/components.js`, change the `missingPropsLabel` component:
```diff
  missingPropsLabel: {
    key: "test",
    name: "Test",
    description: "hello",
    version: "0.0.1",
    props: {
      test: {
-       type: "string",
        description: "test",
      },
    },
  },
```
- At the root of the repo, run the command:
```
npm run test
```

Reply:
Awesome. Fixed the error and bumped the version.